src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<String, Integer> fruitsStorage = new HashMap<>();
```

Comment:
Such approach may cause troubles with maintainability and testing of your code.
Please make it private and add relevant getter.
Getter will be non-static so will require dependency injection of storage via constructor as any other service you rely on